src/bundle/Resources/public/js/scripts/core/suggestion.taggify.js

```diff
@@ -0,0 +1,124 @@
+(function (global, doc, ibexa) {
+    class SuggestionTaggify extends ibexa.core.Taggify {
+        constructor(config) {
+            super(config);
+
+            this.suggestionsListNode = config.suggestionsListNode ?? this.container.querySelector('.ibexa-taggify__suggestions');
+            this.token = config.token ?? doc.querySelector('meta[name="CSRF-Token"]').content;
```

Comment:
We have currently helpers for extracting these, so maybe it can be used here?
```
import { getRestInfo } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
...
const { siteaccess, token } = getRestInfo()

```